playbooks/common/openshift-cluster/openshift_hosted_logging.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
This file will be redundant after https://github.com/openshift/openshift-ansible/pull/5176